src/components/DropdownList.tsx

```diff
@@ -41,10 +43,19 @@ const DropdownLink = styled(Link)`
 		color: ${({ theme }) => theme.colors.primary[0]};
 	}
 `;
-function DropdownList(props: Props) {
+function DropdownList({ items, buttonRef }: Props) {
+	const [top, setTop] = useState(-9999);
```

Comment:
Why -9999?

Reply:
It is definitely arbitrary, but intentionally large.